blocks/comparison/comparison.js

```diff
@@ -0,0 +1,41 @@
+import { getLibs } from '../../scripts/utils.js';
+
+/**
+ * @param {Element} el
+ */
+const init = async (el) => {
+  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+  const rows = Array.from(el?.querySelectorAll(':scope > div'));
+  const headers = Array.from(rows.shift().children);
+
+  const headersRow = createTag('tr');
+  headers.forEach((cell) => {
+    headersRow.appendChild(createTag('th', { scope: 'col' }, cell));
+  });
+  const thead = createTag('thead', {}, headersRow);
+
+  const tbody = createTag('tbody');
+  rows?.forEach((row, idx) => {
+    const bodyRow = createTag('tr');
+    const cells = Array.from(row?.querySelectorAll(':scope > div'));
+    const headerCell = cells.shift();
+
+    if (idx === rows.length - 1 && headerCell.textContent === '') {
+      bodyRow.classList.add('cta-row');
+    }
+    bodyRow.appendChild(createTag('th', { class: '', scope: 'row' }, headerCell));
+    cells.forEach((cell) => {
+      bodyRow.appendChild(createTag('td', {}, cell));
+    });
```

Comment:
Perhaps with something like this, you can append them all at once to reduce the number of DOM updates
```suggestion
    bodyRow.append(...cells.map((cell) => createTag('td', null, cell)));
```

Reply:
Sounds good, thank you